packages/account/src/Configs/trading-assessment-config.js

```diff
@@ -0,0 +1,308 @@
+import { generateValidationFunction, getDefaultFields } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+
+export const trading_assessment_questions = [
+    {
+        question_text: localize(
+            'Do you understand that you could potentially lose 100% of the money you use to trade?'
+        ),
+        section: 'risk_tolerance',
+        answer_options: [
+            { text: localize('Yes'), value: 'Yes' },
+            { text: localize('No'), value: 'No' },
+        ],
+        form_control: 'risk_tolerance',
+        field_type: 'radio',
+    },
+    {
+        question_text: localize('How much knowledge and experience do you have in relation to online trading?'),
+        section: 'source_of_experience',
+        form_control: 'source_of_experience',
+        answer_options: [
+            {
+                text: localize(
+                    'I have an academic degree, professional certification, and/or work experience related to financial services.'
+                ),
+                value: 'I have an academic degree, professional certification, and/or work experience.',
```

Comment:
could you please clarify, we pass these value fields to BE right how they are written here?
why these value and text fields are different for several items here?

Reply:
The value is a key used within BE code, so they have to pass it exactly how it is hardcoded there

Reply:
The text displayed and the value for some dropdown elements are different.